CodeSystem/CodeSystem-Genomics-reason-for-testing.json

```diff
@@ -0,0 +1,103 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "reasonfortesting-genomics",
+  "url": "https://fhir.nhs.uk/CodeSystem/reasonfortesting-genomics",
+  "version": "0.1.0",
+  "name": "ReasonforTestingGenomics",
+  "title": "Reason for Testing Genomics",
+  "status": "draft",
+  "date": "2024-07-09T00:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A set of codes use to identify the reason for requesting a genomics test.",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+  "caseSensitive": true,
+  "content": "complete",
+  "concept": [
+    {
+      "code": "carrier",
+      "display": "Carrier",
+      "definition": "Primary test request"
```

Comment:
Could you update the codesystem so the codes are in a hierarchy? e.g. all the ones you've defined as Primary Test Request are actually children of this concept? You can then update the definition to be more specific to the concept (or remove this entirely).